spaces/domains.json

```
@@ -160,4 +160,5 @@
   "gov.tusker.finance": "tuskerfinance.eth",
   "gov.sunicoin.org": "suni.eth",
   "vote.macaronswap.finance": "macaronswap.eth"
```

You will miss a comma at the end of the line above your sub domain.
```suggestion
  "vote.macaronswap.finance": "macaronswap.eth",
```